vividus/src/main/java/org/vividus/bdd/steps/BddVariableSteps.java

```diff
@@ -177,18 +180,41 @@ public boolean compareVariables(Object variable1, ComparisonRule condition, Obje
      * @param table ExamplesTable with parameters to compare with
      */
     @Then("`$variable` is equal to table ignoring extra columns:$table")
+    @SuppressWarnings("unchecked")
     public void tablesAreEqualIgnoringExtraColumns(Object variable, ExamplesTable table)
     {
-        if (!(variable instanceof Map))
-        {
-            throw new IllegalArgumentException("'variable' should be instance of map");
-        }
-        Map<String, String> expectedMap = MapUtils.convertSingleRowExamplesTableToMap(table);
-        List<EntryComparisonResult> results = ComparisonUtils.checkMapContainsSubMap((Map<?, ?>) variable, expectedMap);
+        isTrue(variable instanceof Map, "'variable' should be instance of map");
+        Map<Object, Object> actualMap = expandListValuesWithSingleElement((Map<Object, Object>) variable);
+
+        List<Parameters> rows = table.getRowsAsParameters(true);
+        isTrue(rows.size() == 1, "ExamplesTable should contain single row with values");
```

Comment:
Why not just table? This sounds like a end-user term whereas ExamplesTable is a class name/developer term

Reply:
ExamplesTable is a part of JBehave terminology, tcase/whitespaces are discussable, but there should be clear ExamplesTable reference

Reply:
End user can guess that we use JBehave only by plugin installed in eclipse, should not we advice end users somewhere in the docs to get acquainted with JBehave by giving some links?

Reply:
we refer ExamplesTable in many variations in docs, so if you'd like to review the usage of this term you can take care

as for current state of the docs: it's a reference book, it's not an ultimate guide